doc/rbd/rbd-kubernetes.rst

```diff
@@ -188,6 +209,7 @@ pool created above, the following YAML file can be used after ensuring that the
         parameters:
            clusterID: b9127830-b0cc-4e34-aa47-9d1a2e9949a8
            pool: kubernetes
+           imageFeatures: layering
```

Comment:
can you also add missing controller expand secrets to this storageclass

https://github.com/ceph/ceph-csi/blob/6a46f381c2f5023862d9be80515b41abd273c8bd/examples/rbd/storageclass.yaml#L59-L60

Reply:
It's only needed for volume expansion, right?  And we don't set `allowVolumeExpansion` in this example.

I guess there is no reason to hide that functionality though.  I'll add both.

Reply:
Yes, correct. Expansion is supported from long time.